docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -3,21 +3,22 @@ sidebar_position: 3
 ---
 # Deployment Manager configuration
 
-Configuration of Deployment Manager, which is component of the Designer that deploys scenario to given Engine (e.g. Lite or Flink). 
-Type of Deployment Manager is defined with `type` parameter, e.g. for running scenarios with Flink streaming job we would configure: 
+Deployment Manager deploys scenarios from the Designer to the engine on which scenarios are processed. Check [configuration areas](./#configuration-areas) to understand where Deployment Manager configuration should be placed in Nussknacker configuration.
+
+Below you can find a snippet of Deployment Manager configuration. 
 ```
 deploymentConfig {     
   type: "flinkStreaming"
   restUrl: "http://localhost:8081"
   # additional configuration goes here
 }
 ```
+`type` parameter determines engine to which the scenario is deployed. Currently there are only two values allowed: `flinkStreaming` and `lite-k8s`. The `type` parameter is set in both the minimal working configuration file (Docker image and binary distribution) and Helm chart - you will need to set it on your won. 
```

Comment:
There are also embedded types: `request-response-embedded`, `streaming-lite-embedded`, though they're not heavy-docummented. What's more, they can be customizable with code

Reply:
Removed the fragment with allowed values, corrected the meaning ("not" was missing) -> _you will NOT need to ..._